appveyor.yml

```diff
@@ -172,8 +170,11 @@ for:
     - python -m pip install git+https://github.com/frerich/clcache.git
 
   before_build:
+    - FOR /D %%G IN (%WINLIB_PATH%\Qt-*) DO SET QT_PATH=%%G
```

Comment:
...and some people say the unix command line is complicated.

Reply:
Hahahah! Yes , for loops in windows shell are a bit complicated. More so because depending on the parameter ( /D in this case ) it does different things. 
This concrete line can be understood as: for each directory in winlibpath that starts with Qt-, set QT_PATH to that absolute path 
( https://ss64.com/nt/for_d.html )